tools/vuln_valid/index.js

```diff
@@ -1,79 +1,7 @@
-'use strict';
-const joi = require('joi').extend(require('joi-extension-semver'));
-const path = require('path');
-const fs = require('fs');
+'use strict'
 
+const vulnValidate = require('./vulnValidate')
```

Comment:
nit, add semicolon

in general, I'll probably add later an eslint/prettier setup so we don't have this issue in the future of unaligned code style

Reply:
OKay. I'll fix that. I could add it to this PR itself if that sounds good to you.